modules/ROOT/pages/slsa.adoc

```diff
@@ -0,0 +1,130 @@
+
+= Red Hat Trusted Application Pipeline, Enterprise Contract, and SLSA
+
+
+.Overview
+****
+
+Red Hat Trusted Application Pipeline (RHTAP) is an industry leading cloud build
+system that aims to meet link:https://slsa.dev/spec/v1.0/levels[SLSA Build L3].
```

Comment:
Is it worth calling out that we're targeting Build L3? or could we keep it more generic and say we're looking to meet or exceed the SLSA spec. By linking directly to Build L3, that ties to a specific definition which is true today, but may not be true later, and remembering to come back and update this link if that changes seems ambitious.

Reply:
I used that wording since I thought it was one of the marketing points. I used the softer "aims to meet" rather than "meets" in case we aren't quite sure yet. Are you suggesting something like "aims to meet or exceed SLSA Build L3"? Or something else?